lib/xcpretty/formatters/formatter.rb

```diff
@@ -38,7 +38,7 @@ def format_passing_test(suite, test, time);                EMPTY; end
     def format_pending_test(suite, test);                      EMPTY; end
     def format_measuring_test(suite, test, time);              EMPTY; end
     def format_failing_test(suite, test, reason, file_path);   EMPTY; end
-    def format_process_pch(file, build_target);                EMPTY; end
+    def format_process_pch(cpp, file, build_target);           EMPTY; end
     def format_process_pch_command(file_path, build_target);   EMPTY; end
```

Comment:
Also, if we are talking performance: the regex that matches this is one of the worst ones in terms of performance (my 80k step regex before we fixed it was based on it), which means that even though we don't use it, since we left it in it causes a hit to performance. It might be a good idea to remove it after all. Might also be a good idea to go over the other regexes and optimize them, and also remove everything we don't use, since the parser still attempts to match even the regexes that we don't use for simple formatter and removing them might also improve performance.

Reply:
Spoke to Barak and removed it.